### PR TITLE
[wip] anchore-policy-validator: add optional allowAll whitelist

### PR DESCRIPTION
**Pull request by orymate**

### CODE REVIEW

File: anchore-policy-validator/templates/whitelist.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{{if .Values.allowAll}}
+apiVersion: security.banzaicloud.com/v1alpha1
+kind: whitelistitems.security.banzaicloud.com
```
Comment (pbalogh-sa) [inline anchore-policy-validator/templates/whitelist.yaml]: kind: WhiteListItem

**Comment by pbalogh-sa** [comment]
Please update chart version.

File: anchore-policy-validator/templates/whitelist.yaml (hunk 1 above)
Comment (orymate) [reply anchore-policy-validator/templates/whitelist.yaml]: both are ok here

**Comment by orymate** [comment]
Superseded by banzaicloud/pipeline#1496

**Pull request closed without merging**